src/main/java/org/javarosa/core/model/TriggerableDag.java

```diff
@@ -168,15 +168,8 @@ public TriggerableDag(EventNotifierAccessor accessor) {
 
             try {
 
-                // Now identify all of the fully qualified nodes which this
-                // triggerable updates. (Multiple nodes can be updated by the same trigger)
-                List<TreeReference> qualifiedList = evalContext.expandReference(contextRef);
```

Comment:
Think through what it would mean for `contextRef` to not be fully qualified.

Reply:
If the reference is not fully qualified, then the `EvaluationContext` that we create later won't be able to resolve relative paths, which would be bad. They need to be absolute too.

Fortunately, we know that all trigger refs arriving to `evaluateTriggerable` are absolute and fully qualified.

In any case, I think I want to study this just a little bit more to be on the safe side. We can always conditionally expand the trigger reference if we detect that it's ambiguous (there's a `TreeReference.isAmbiguous()` method). If the trigger ref is relative, we can also contextualize it with the triggerable too.

Both things are what we were doing before and I removed in this PR. The difference would be that now we would be doing it conditionally and giving enough context to future readers. It would feel bad, though, to not be able to explain precisely under which scenarios that could happen.